src/lib/gssapi/spnego/gssapiP_spnego.h

```diff
@@ -49,6 +49,11 @@ extern "C" {
 #define	ERR_SPNEGO_NEGOTIATION_FAILED		0x20000004
 #define	ERR_SPNEGO_NO_TOKEN_FROM_ACCEPTOR	0x20000005
 
+
+#define SPNEGOINT_CHK_EMPTY(p) ((p != NULL) \
```

Comment:
Okay, I see the problem.  SPNEGO potentially creates several mechglue contexts as it tries different mechs, and passing through the create_sec_context/set_context_flags call before init/accept_sec_context would only work if we created at most one mechglue context.  You can ignore this comment.